Rust/build.rs

```diff
@@ -8,130 +8,108 @@
  * DEVELOPER: Mouri_Naruto (Mouri_Naruto AT Outlook.com)
  */
 
-fn main()
-{
-    if !std::env::var("CARGO_CFG_WINDOWS").is_ok()
-    {
-        println!("cargo:warning=VC-LTL only supports Windows host. So the VC-LTL stage is skipped.");
+fn main() {
+    if !std::env::var("CARGO_CFG_WINDOWS").is_ok() {
+        println!(
+            "cargo:warning=VC-LTL only supports Windows host. So the VC-LTL stage is skipped."
+        );
         return;
     }
 
-    let target_os = std::env::var("CARGO_CFG_TARGET_OS").expect(
-        "CARGO_CFG_TARGET_OS should be set by cargo.");
-    let target_env = std::env::var("CARGO_CFG_TARGET_ENV").expect(
-        "CARGO_CFG_TARGET_ENV should be set by cargo.");
+    let target_os =
+        std::env::var("CARGO_CFG_TARGET_OS").expect("CARGO_CFG_TARGET_OS should be set by cargo.");
+    let target_env = std::env::var("CARGO_CFG_TARGET_ENV")
+        .expect("CARGO_CFG_TARGET_ENV should be set by cargo.");
 
-    if !target_os.contains("windows")
-    {
-        println!("cargo:warning=VC-LTL only supports Windows target. So the VC-LTL stage is skipped.");
+    if !target_os.contains("windows") {
+        println!(
+            "cargo:warning=VC-LTL only supports Windows target. So the VC-LTL stage is skipped."
+        );
         return;
     }
 
-    if !target_env.contains("msvc")
-    {
-        println!("cargo:warning=VC-LTL only supports MSVC toolchain. So the VC-LTL stage is skipped.");
+    if !target_env.contains("msvc") {
+        println!(
+            "cargo:warning=VC-LTL only supports MSVC toolchain. So the VC-LTL stage is skipped."
+        );
         return;
     }
 
-    let package_root = std::env::var("CARGO_MANIFEST_DIR").expect(
-        "CARGO_MANIFEST_DIR should be set by cargo.");
-    let target_arch = std::env::var("CARGO_CFG_TARGET_ARCH").expect(
-        "CARGO_CFG_TARGET_ARCH should be set by cargo.");
+    let vcltl_root = std::env::var("VCLTL_ROOT")
+        .or_else(|_| std::env::var("CARGO_MANIFEST_DIR"))
+        .expect("VCLTL_ROOT or CARGO_MANIFEST_DIR should be set.");
+    let target_arch = std::env::var("CARGO_CFG_TARGET_ARCH")
+        .expect("CARGO_CFG_TARGET_ARCH should be set by cargo.");
 
-    let target_platform;
-    let mut minimum_version = 6000;
-
-    if target_arch.contains("x86_64")
-    {
-        target_platform = "x64";
-        if minimum_version < 3790
-        {
-            minimum_version = 3790;
-        }
-    }
-    else if target_arch.contains("x86")
-    {
-        target_platform = "Win32";
-        if minimum_version < 2600
-        {
-            minimum_version = 2600;
-        }
-    }
-    else if target_arch.contains("arm")
-    {
-        target_platform = "ARM";
-        if minimum_version < 9200
-        {
-            minimum_version = 9200;
+    let version = match std::env::var("VCLTL_VERSION") {
+        Ok(version) => version.parse::<usize>().unwrap(),
+        _ if cfg!(feature = "win7") => 9200,
```

Comment:
9200 is Windows 8？

Reply:
I'm not sure either, so what's the actual value for Windows 7?

Reply:
Windows 7 RTM is Build 7600

Reply:
> I'm not sure either, so what's the actual value for Windows 7?

Windows 7 RTM is 7600?